packages/main/src/Slider.js

```diff
@@ -0,0 +1,271 @@
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
+import SliderTemplate from "./generated/templates/SliderTemplate.lit.js";
+import {
+	isDown,
+	isUp,
+	isTabNext,
+	isTabPrevious,
+} from "../../base/src/events/PseudoEvents.js";
+
+// Styles
+import SliderCss from "./generated/themes/Slider.css.js";
+
+/**
+ * @private
+ */
+const metadata = {
+	tag: "ui5-slider",
+	properties: /** @lends sap.ui.webcomponents.main.Slider.prototype */ {
+		/**
+		 * Defines whether the <code>ui5-slider</code> is disabled
+		 * (default is set to <code>false</code>).
+		 * A disabled <code>ui5-slider</code> can't be pressed or
+		 * focused, and it is not in the tab chain.
+		 *
+		 * @type {boolean}
+		 * @defaultvalue false
+		 * @public
+		 */
+		disabled: {
+			type: Boolean,
+		},
+
+		/**
+		 * Defines the currently selected value
+		 * @type {string}
+		 * @defaultvalue ""
+		 * @public
+		 */
+		value: {
+			type: String,
+		},
+
+		/**
+		 * Defines the label of the slider.
+		 * @type {string}
+		 * @defaultvalue ""
+		 * @public
+		 */
+		label: {
+			type: String,
+			defaultValue: "",
+		},
+
+		/**
+		 * Indicates if the slider is expanded.
+		 * @type {boolean}
+		 * @defaultvalue false
+		 * @private
+		 */
+		_expanded: {
+			type: Boolean,
+		},
+
+		_items: {
+			type: Object,
+		},
+	},
+	slots: /** @lends sap.ui.webcomponents.main.Slider.prototype */ {
+
+	},
+	events: /** @lends sap.ui.webcomponents.main.Slider.prototype */ {
+		/**
+		 * Fires when the slider is expanded.
+		 */
+		expand: {},
+
+		/**
+		 * Fires when the slider is collapsed.
+		 */
+		collapse: {},
+
+		/**
+		 *  Fires when new value is selected.
+		 */
+		valueSelect: {
+			value: {
+				type: String,
+			},
+		},
+	},
+};
+
+/**
+ * @class
+ *
+ * <h3 class="comment-api-title">Overview</h3>
+ *
+ *
+ * <h3>Usage</h3>
+ *
+ * For the <code>ui5-slider</code>
+ * <h3>ES6 Module Import</h3>
+ *
+ * <code>import @ui5/webcomponents/dist/Slider.js";</code>
+ *
+ * @constructor
+ * @author SAP SE
+ * @alias sap.ui.webcomponents.main.Slider
+ * @extends UI5Element
+ * @tagname ui5-slider
+ * @public
+ */
+class Slider extends UI5Element {
+	static get metadata() {
+		return metadata;
+	}
+
+	static get render() {
+		return litRender;
+	}
+
+	static get styles() {
+		return SliderCss;
+	}
+
+	static get template() {
+		return SliderTemplate;
+	}
+
+	constructor() {
+		super();
+		this._currentElementIndex = 0;
+		this._itemCellHeight = 0;
+	}
+
+	onBeforeRendering() {
+		this._updateItemCellHeight();
+	}
+
+	_updateItemCellHeight() {
+		this._itemCellHeight = this.shadowRoot.querySelectorAll(".ui5-slider-item").length && Number(getComputedStyle(this.shadowRoot.querySelector(".ui5-slider-item")).getPropertyValue("--_ui5_slider_item_height").replace("rem", ""));
+	}
+
+	onAfterRendering() {
+		this.shadowRoot.querySelector(".ui5-slider-wrapper").addEventListener("wheel", e => {
```

Comment:
Can you subscribe for the event in the Slider.hbs file

Reply:
Done